content/departments/tech-ops/tools/computer-setup.md

```diff
@@ -1,28 +1,27 @@
-## Computer requirements
+# Computer requirements
```

Comment:
I think this page needs to get moved to company wide part of handbook `company-info-and-processess/policies` instead of being under tech-ops.

Reply:
Agreed

Reply:
I'm not sure how I move this so I'll commit the changes and then do this in a separate PR (still learning here!)